Example/Podfile

```diff
@@ -1,5 +1,5 @@
 use_frameworks!
-platform :ios, '9.0'
+platform :ios, '13.0'
```

Comment:
With this change, we will drop support of iOS versions prior to iOS 13, right?

Reply:
Glad you asked. I had changed that to require iOS 13 then went back and fixed it where it could still be compatible with 9+ and forgot to change the required version back. I pushed a new commit to update that back to 9.0. The changes I made to support the newer devices would require iOS 13+ but those devices were shipped with OS higher than 13 and I did wrap the changes in if #available(iOS 13.0, *) so it should be fine.